SplitIt/app/src/main/java/project/cs495/splitit/MemberAddDialog.java

```diff
@@ -0,0 +1,110 @@
+package project.cs495.splitit;
+
+import android.app.Dialog;
+import android.content.Context;
+import android.content.Intent;
+import android.util.Log;
+import android.view.KeyEvent;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.ImageButton;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.database.DataSnapshot;
+import com.google.firebase.database.DatabaseError;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+import com.google.firebase.database.Query;
+import com.google.firebase.database.ValueEventListener;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import project.cs495.splitit.models.Group;
+import project.cs495.splitit.models.User;
+
+public class MemberAddDialog extends Dialog{
```

Comment:
We had been using DialogFragments, but if this works it should be fine.

Reply:
We were using DialogFragments because the dialogs were within a fragment on the tabbed pages.  This is in its own non-tabbed page, so Dialog is used.